src/style.css

```diff
@@ -13,7 +13,6 @@ body {
 
 .wrapper {
   width: 50rem;
-  min-height: 30vh;
   margin-left: 7rem;
   margin-top: 90px;
   border-radius: 10px;
```

Comment:
- [ ] [OPTIONAL] I think it could be good to give a bit of style to your page since for now, it does not really look good 😄  this is how it looks or now 👇 
![image](https://user-images.githubusercontent.com/55465133/135724573-a2e62f1d-743e-4096-bb66-90f95206ec00.png)